include/vcpkg/base/messages.h

```diff
@@ -590,6 +590,15 @@ namespace vcpkg
                     "-- Automatically setting {env_var} environment variables to \"{url}\".");
     DECLARE_MESSAGE(AvailableArchitectureTriplets, (), "", "Available architecture triplets:");
     DECLARE_MESSAGE(AvailableHelpTopics, (), "", "Available help topics:");
+    DECLARE_MESSAGE(BaselineFileNoDefaultField,
+                    (msg::value),
+                    "{value} is a commit sha",
```

Comment:
The other ones say `"{value} is a 40-digit hexadecimal SHA"`. This seems to be used a lot, should we add a specific named replacement for it?